psi4/driver/driver_cbs_helper.py

```diff
@@ -0,0 +1,649 @@
+            cbsscheme += """   Alpha (exponent) Value:           % 16.12f\n""" % (alpha)
+            cbsscheme += """   Beta (coefficient) Value:         % 16.12f\n\n""" % (beta)
+
+            name_str = "%s/(%s,%s)" % (functionname.upper(), _zeta_val2sym[zLO].upper(), _zeta_val2sym[zHI].upper())
+            cbsscheme += """   @Extrapolated """
+            cbsscheme += name_str + ':'
+            cbsscheme += " " * (18 - len(name_str))
+            cbsscheme += """% 16.12f\n\n""" % value
+            core.print_out(cbsscheme)
+            logger.debug(cbsscheme)
+
+        return value
+
+    elif isinstance(valueLO, np.ndarray):
+        beta = (valueHI - valueLO) * beta_division
+        value = valueHI - beta * beta_mult
+
+        if verbose > 2:
+            cbsscheme = f"""\n   ==> Truhlar 2-point power SCF extrapolation for method: {functionname.upper()} <==\n"""
+            cbsscheme += f"""\n   LO-zeta ({zLO}) Data\n"""
+            cbsscheme += nppp(valueLO)
+            cbsscheme += f"""\n   HI-zeta ({zHI}) Data\n"""
+            cbsscheme += nppp(valueHI)
+
+            cbsscheme += """   Alpha (exponent) Value:          %16.8f\n""" % (alpha)
+            cbsscheme += f"""\n   Beta Data\n"""
+            cbsscheme += nppp(beta)
+            cbsscheme += f"""\n   Extrapolated Data\n"""
+            cbsscheme += nppp(value)
+            core.print_out(cbsscheme)
+            logger.debug(cbsscheme)
+
+        return value
+
+    else:
+        raise ValidationError("scf_xtpl_truhlar_2: datatype is not recognized '%s'." % type(valueLO))
+
+
+def scf_xtpl_karton_2(functionname, zLO, valueLO, zHI, valueHI, verbose=1, alpha=None):
+    r"""Extrapolation scheme using root-power form for reference energies with two adjacent
+    zeta-level bases. Used by :py:func:`~psi4.cbs`.
+
+    Parameters
+    ----------
+    functionname : str
+        Name of the CBS component.
+    zLO : int
+        Lower zeta level.
+    valueLO : float
+        Lower value used for extrapolation.
+    zHI : int
+        Higher zeta level. Should be equal to zLO + 1.
+    valueHI : float
+        Higher value used for extrapolation.
+    alpha : float, optional
+        Overrides the default :math:`\alpha = 6.3`
+
+    Returns
+    -------
+    float
+        Returns :math:`E_{total}^{\infty}`, see below.
+
+    Notes
+    -----
+    The extrapolation is calculated according to [3]_:
+    :math:`E_{total}^X = E_{total}^{\infty} + \beta e^{-\alpha\sqrt{X}}, \alpha = 6.3`
+
+    References
+    ----------
+
+    .. [3] Karton, Martin, Theor. Chem. Acc. 115 (2006) 330-333,
+       DOI: 10.1007/s00214-005-0028-6
+
+    """
+
+    if type(valueLO) != type(valueHI):
+        raise ValidationError(
+            f"scf_xtpl_karton_2: Inputs must be of the same datatype! ({type(valueLO)}, {type(valueHI)})")
+
+    if alpha is None:
+        alpha = 6.30
+
+    beta_division = 1 / (math.exp(-1 * alpha) * (math.exp(math.sqrt(zHI)) - math.exp(math.sqrt(zLO))))
```

Comment:
This should be `1 / (math.exp(-1 * alpha * math.sqrt(zHI)) - math.exp(-1 * alpha * math.sqrt(zLO)))` as exponents don't factorize out like that. I'm not sure how long this bug has been in `cbs()`, whoops!